extended/src/main/java/apoc/vectordb/VectorMappingConfig.java

```diff
@@ -15,6 +15,7 @@ enum MappingMode {
     public static final String EMBEDDING_KEY = "embeddingKey";
     public static final String SIMILARITY_KEY = "similarity";
     public static final String MODE_KEY = "mode";
+    public static final String NO_FIELDS_ERROR_MSG = "You have to define `field` list of parameter to be returned";
```

Comment:
"You need to define either the 'field' list parameter or the 'metadataKey' string parameter"